src/storage/store/string_column_chunk.cpp

```diff
@@ -108,10 +108,7 @@ void StringColumnChunk::write(const Value& val, uint64_t posToWrite) {
 }
 
 void StringColumnChunk::setValueFromString(const char* value, uint64_t length, uint64_t pos) {
-    if (length > BufferPoolConstants::PAGE_4KB_SIZE) {
-        throw CopyException(
-            ExceptionMessage::overLargeStringValueException(std::to_string(length)));
-    }
+    TableCopyUtils::validateStrLen(length);
```

Comment:
Let's add a test for this function.

Reply:
This function is only used for copyrel, and will be removed by @ray6080 quickly once the storage layout is changed.